doc/QuickStart.md

```diff
@@ -121,9 +159,16 @@ or in the `#pbot2` channel on the Freenode network.
 Starting PBot
 -------------
 
-### Usage
+### Usage *(pbot)*
+
+pbot [directory overrides...; e.g. data_dir=...] [registry overrides...; e.g. irc.botnick=...]
+
+    $ screen -dmS pbot /home/pbot/pbot/pbot data_dir=/home/pbot/pbot/freenode
+    $ screen -r pbot 	(for attaching to the screen named: pbot)
+    ctrl+a than ctrl+d 	(for detaching without closing the screen)
 
-    pbot [directory overrides...; e.g. data_dir=...] [registry overrides...; e.g. irc.botnick=...]
+> or $ screen -dmS coolbot /home/pbot/pbot/pbot data_dir=/home/pbot/pbot/coolbot
+> or $ ./pbot data_dir=/home/pbot/pbot/...
```

Comment:
Users are expect to understand how they prefer to launch programs such as PBot. Most users probably prefer `tmux` over `screen` these days, anway. In any case, I prefer not to recommend any specific way to start PBot since users are expected to be familiar with their operating system of choice (Linux, Windows, etc) which support Perl. These system may not have `screen`, etc. Users are expected to understand their operating system.